au.com.hpcoders.Minsky.desktop

```diff
@@ -1,9 +1,10 @@
 [Desktop Entry]
 Name=Minsky
-Exec=minsky %U
+Exec=minsky %f
 Terminal=false
 Type=Application
-Icon=minsky
+Icon=au.com.hpcoders.Minsky
```

Comment:
I'd prefer to use net.sf.minsky as the domain properly belongs to minsky, rather than my company. But we can leave this for now.

Reply:
This kind of app ID does not need to necessarily match the website domain of the app. Usually the author is prefixed to the app name, hence why I did it this way. It is how it is usually done in macOS (to my knowledge) and in modern Freedesktop etiquette (it is almost in the Flatpak world, in which I plan to package Minsky). If you still want to have it with the website domain, I can send a PR with the required modifications.